### PR TITLE
Initial contribution MAX! Binding Tests

### PR DESCRIPTION
**Pull request by marcelrv**
Signed-off-by: Marcel Verpaalen <marcel@verpaalen.com>

### CODE REVIEW

File: addons/binding/org.openhab.binding.max.test/pom.xml [hunk 1]
```diff
@@ -0,0 +1,115 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+	<modelVersion>4.0.0</modelVersion>
+		 <groupId>org.openhab.binding</groupId>
+	<artifactId>org.openhab.binding.max.test</artifactId>
+	<packaging>eclipse-test-plugin</packaging>
+	<name>openHAB MAX! Binding Tests</name>
```
Comment (kaikreuzer) [inline addons/binding/org.openhab.binding.max.test/pom.xml]: like the bindings, the name should be without "openHAB" here, i.e. simply "MAX! Binding Tests"

File: targetplatform/openhab.target [hunk 2]
```diff
@@ -1,117 +1,152 @@
...
+<unit id="org.eclipse.jdt.feature.group" version="3.10.0.v20140606-1536"/>
+<repository location="http://download.eclipse.org/releases/luna/201406250900/"/>
+</location>
+<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="slicer" includeSource="true" type="InstallableUnit">
+<unit id="org.eclipse.xtend.sdk.feature.group" version="2.6.2.v201407030533"/>
+<unit id="org.eclipse.xtext.sdk.feature.group" version="2.6.2.v201407030533"/>
+<repository location="http://download.eclipse.org/modeling/tmf/xtext/updates/releases/maintenance/R201407030533"/>
+</location>
+<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="slicer" includeSource="true" type="InstallableUnit">
+<unit id="org.eclipse.emf.mwe2.runtime.sdk.feature.group" version="2.6.1.v201406110919"/>
```
Comment (kaikreuzer) [inline targetplatform/openhab.target]: I think this is not required and can be removed.

File: targetplatform/openhab.target [hunk 3]
```diff
@@ -1,117 +1,152 @@
...
+<repository location="http://www.openhab.org/jupnp/update-site"/>
+</location>
+<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="slicer" includeSource="true" type="InstallableUnit">
+<unit id="org.eclipse.jetty.bundles.f.feature.group" version="9.2.1.201406101620"/>
+<unit id="org.eclipse.jetty.bundles.f.source.feature.group" version="9.2.1.201406101620"/>
+<repository location="http://download.eclipse.org/jetty/updates/jetty-bundles-9.x/9.2.1.v20140609"/>
+</location>
+<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="slicer" includeSource="true" type="InstallableUnit">
+<unit id="org.eclipse.equinox.sdk.feature.group" version="3.10.0.v20140606-1602"/>
+<unit id="org.eclipse.jdt.feature.group" version="3.10.0.v20140606-1536"/>
```
Comment (kaikreuzer) [inline targetplatform/openhab.target]: I think this is not required and can be removed.

File: targetplatform/openhab.target [hunk 4]
```diff
@@ -1,117 +1,152 @@
...
+<unit id="org.slf4j.log4j.source" version="1.7.2.v20130115-1340"/>
+<repository location="http://download.eclipse.org/tools/orbit/downloads/drops/R20140525021250/repository/"/>
+</location>
+<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="slicer" includeSource="true" type="InstallableUnit">
+<unit id="org.eclipse.emf.sdk.feature.group" version="2.10.0.v20140519-0339"/>
+<repository location="http://download.eclipse.org/modeling/emf/emf/updates/2.10/core/R201405190339"/>
+</location>
+<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="slicer" includeSource="true" type="InstallableUnit">
+<unit id="org.codehaus.groovy23.feature.feature.group" version="2.9.1.xx-201411061335-e44-RELEASE"/>
+<unit id="org.codehaus.groovy.eclipse.feature.feature.group" version="2.9.1.xx-201411061335-e44-RELEASE"/>
```
Comment (kaikreuzer) [inline targetplatform/openhab.target]: I think this is not required and can be removed.

**Comment by kaikreuzer** [comment]
Thanks for the tests, Marcel!
I just tried to minimise the additions to the TP - in my tests, it works fine both in the IDE and with Maven when leaving out the three features.
Will have to manage to get Groovy into the Yoxos profile soon - just asked Yoxos support for help, so let's see...

**Comment by marcelrv** [comment]
Indeed, still works when these lines are taken out.
Thx

**Pull request merged**